server/api/src/services/nc.py

```diff
@@ -0,0 +1,26 @@
+import db
+import pandas as pd
+
+
+def get_ncs():
+    """
+    Gets a sorted list of all NCs in the data
+    """
+    df = pd.read_sql("""
+        select distinct nc from requests
+        where nc is not null
+        order by nc
+    """, db.engine)
+    return df['nc'].tolist()
+
+
+def get_request_types():
+    """
+    Gets a sorted list of all request types in the data
+    """
+    df = pd.read_sql("""
+        select distinct requesttype from requests
+        where requesttype is not null
+        order by requesttype
+    """, db.engine)
+    return df['requesttype'].tolist()
```

Comment:
Good call on starting to build out an NC service. I could def see it being useful.

Reply:
i'll keep request types here for now but they could probably go into another file too. incidentally, i think at some point we should explore converting the types to a look up table. it's a little awkward that these are strings. i don't know if you saw my other note, but it looks like the 'Report Water Waste' type is missing from the UI. would be good to have an endpoint where the client gets these types and then filters by ID instead of string

Reply:
Yep, that 'Report Water Waste' has never been part of the UI since I got here. Not sure if that was a decision made at some point or whether it's a mistake.

Reply:
interesting. not sure whether to account for it in tests then. will give that some thought...

Reply:
ok cool. btw I also agree about converting the types from strings. In fact at one point I was hoping we could have a `/metadata` endpoint that would serve up all the "domain knowledge" -- NC ids, NC shapes, request types, etc. -- so that the client wouldn't need to hard code all of that info in various geojson/constants files. But then that info got so entangled in the frontend codebase that I dropped it. But long term we can do a lot better in terms of making the backend a single point of truth about the structure of our data.